packages/query/src/stacks/stacks-client.ts

```diff
@@ -39,6 +39,26 @@ export interface CallReadOnlyFunctionArgs {
   signal?: AbortSignal;
 }
 
+const PRIORITY_LEVELS = {
```

Comment:
```suggestion
const priorityLevels = {
```

no need for upper casing here

Reply:
What if we need different priorities in web and mobile?

Reply:
It could yeah. Imagine this:

```ts
// mobile
const query = useSomeQuery(createQueryConfig({ ...args, priority: 'high' }))
// web
const query = useSomeQuery(createQueryConfig({ ...args, priority: 'low' }))
```

Reply:
We can choose a scale of whatever granularity we need, so we can add very-high, or use numbers. Important thing is each platform decides.

> also managing request priorities when they are in request itself is quite painful

Agree. I'm not really sure we want to go too far down the path of juggling request priorities. What is most important isn't fixed per app, it varies depending on route/what the users doing etc, so while setting a priority can be helpful, maybe we shouldn't rely on it too much 🤹🏼